src/archivist/filter/index.js

```diff
@@ -20,6 +20,18 @@ const { CiceroMarkTransformer } = ciceroMark;
 
 const ciceroMarkTransformer = new CiceroMarkTransformer();
 
+/**
+ * Filter content and convert it to markdown
```

Comment:
What is “content”? 🙂 

```suggestion
 * Filter content and convert it to Markdown
```

Reply:
I think it is worth explaining that `content` can be either a DOM dump as a `string` or a binary (`blob`? `buffer`?) containing PDF data. Otherwise, as a user of this library, I have no idea what form of “content” I'm supposed to give it 🙂

Reply:
Added `Content to filter: a buffer containing PDF data in case mimetype associated is PDF or a DOM dump of an HTML page given as a string`